rho.spec

```diff
@@ -59,14 +59,17 @@ cp -rp roles %{buildroot}%{_datadir}/ansible/%{name}/
 %{_datadir}/ansible/%{name}/roles/*
 
 %changelog
-* Thu Feb 15 2018 Christopher Hambridge <chambrid@redhat.com> 0.0.32-2
+* Wed Mar 21 2018 Noah Lavine <nlavine@redhat.com> 0.0.33-1
+- Bug fix to Ansible task bash globbing (nlavine@redhat.com)
+- Bug fix for unicode truncation (chambrid@redhat.com)
+
+* Thu Feb 15 2018 Christopher Hambridge <chambrid@redhat.com> 0.0.32-6
```

Comment:
Question: why did this change from `0.0.32-2` to `0.0.32-6`?

Reply:
Due to issues with the actual builds in the fedora Dist Git. Our git was out of synch.